features/other_b/support.feature

```diff
@@ -45,3 +45,4 @@ Feature: Filing a support request
     And I press "Send"
   Then I should see "Your message was sent to the Archive team - thank you!"
     And 2 emails should be delivered
+    
```

Comment:
*scratches head* I'm not sure why the newline is being so uncooperative here. We can live with it, but it might be worth Googling the name of your text editor or IDE + "newline end of file" or similar to get it straightened out if you plan on making any more pull requests.

Reply:
_ugh_ I thought I'd disabled my tooling for that. Not sure how it crept in again - maybe it somehow got enabled again. I'll check again. Good spot.